bindings/python/README.md

```diff
@@ -70,6 +77,12 @@ print(f'{node_info}')
 
 ## Wallet Usage
 
+The following example will create a
+new [`Wallet`](https://wiki.iota.org/shimmer/iota-sdk/references/python/iota_sdk/wallet/) [`Account`](https://wiki.iota.org/shimmer/iota-sdk/references/python/iota_sdk/wallet/account/)
+that connects to the [Shimmer Testnet](https://api.testnet.shimmer.network) using the
+[`StrongholdSecretManager`](https://wiki.iota.org/shimmer/iota-sdk/references/python/iota_sdk/secret_manager/#strongholdsecretmanager-objects)
```

Comment:
Link returns `Page Not Found`

Reply:
Actually all the links don't work, will the wiki be updated in the next days so they work?

Reply:
Yeah they decided to use the future links to avoid having us to change it again after release